packages/codemods/src/codeMods/__tests__/ComponentToCompat/ComponentToCompat.test.ts

```diff
@@ -0,0 +1,84 @@
+import { Project } from 'ts-morph';
+import * as Root from '../mock/compat/mockIndex';
+import {
+  IRawCompat,
+  runComponentToCompat,
+  getNamedExports,
+  buildCompatHash as buildHash,
+  repathNamedImports,
+  repathPathedImports,
+} from '../../mods/ComponentToCompat/CompatHelpers';
+import * as Button from 'office-ui-fabric-react/lib-commonjs/Button';
```

Comment:
Why are you importing from `lib-commonjs`?

Reply:
This is only runs via node and common-js is the only type of import that is supported.

Reply:
Typically we import from `lib` anyway and handle the remapping with `moduleNameMapper` settings (which should be configured automatically after a recent change I did)

Reply:
I'm going to leave it as commonjs for now, I can test removing that later once this has been checked in.